roles/manage-aws-infra/templates/casl-aws-cformation.yml.j2

```diff
@@ -428,6 +451,46 @@ Resources:
             DeleteOnTermination: "true"
             VolumeSize: "{{ etcd_docker_volume_size }}"
 {% endfor %}
+{% for idx in range(0, aws_num_bastion | int) %}
```

Comment:
Please review this loop ({% for idx in range(0, aws_num_bastion | int) %}) as you are adding bastion instances here but there are references to the etcd instance type variables. I think you forgot to adapt this block after copy-pasting from previous block